yarn-project/archiver/src/archiver/archiver_store.ts

```diff
@@ -167,12 +128,12 @@ export interface ArchiverDataStore {
    * Gets the number of the latest L2 block processed.
    * @returns The number of the latest L2 block processed.
    */
-  getBlockNumber(): Promise<number>;
+  getSynchedL2BlockNumber(): Promise<number>;
 
   /**
-   * Gets the last L1 block number processed by the archiver
+   * Gets the synch point of the archiver
    */
-  getL1BlockNumber(): Promise<ArchiverL1SynchPoint>;
+  getSynchedL1BlockNumbers(): Promise<ArchiverL1SynchPoint>;
```

Comment:
Found the naming here confusing so I cleaned that up.

Reply:
Found the new naming a bit confusing as well 😅 naming is hard.